gitopsaddon/charts/openshift-gitops-operator/templates/crds/appprojects.argoproj.io.crd.yaml

```
@@ -1,6 +1,8 @@
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
```

YAMLlint (1.38.0)

[error] empty value in block mapping

(empty-values)